plugins/vengeance-helper

```diff
@@ -1,2 +1,2 @@
-repository=https://github.com/Th3mike/Vengeance-Helper.git
-commit=c3c63d5f363092fe2ec90baf067b815aeca6011c
+repository=https://github.com/cwjoshuak/runelite-plugins/tree/vengeance-helper
```

Comment:
```suggestion
repository=https://github.com/cwjoshuak/runelite-plugins.git
```

Reply:
@Felanbird I had to change the repo url to a forked one for the diff to go through, and i messed up the first rebase. Should be fixed now